turbinia/config/turbinia_config.py

```diff
@@ -16,20 +16,57 @@
 from __future__ import unicode_literals
 
 
-"""Dummy Turbinia config file."""
+"""Turbinia Config Template"""
 
-# Turbinia Config
+##########
+# COMMON #
+##########
 
-# 'PSQ' is currently the only valid option
+# Which user account Turbinia runs as
+USER = 'turbinia'
+
+# Turbinia's installion directory
+TURBINIA_DIR = '/opt/turbinia'
+
+# Turbinia source code directory (local git repository)
+SRC_DIR = '%s/src' % TURBINIA_DIR
+
+# GCSFuse mount for Turbinia scripts & logging and Google Cloud Functions
+MOUNT_POINT = '/mnt/turbinia'
+
+# Turbinia CLI
+TURBINIACTL = '%s/turbinia/turbiniactl' % SRC_DIR
+
+# Virtualenv directory
+TURBINIAENV = '%s/turbinia-env' % HOME_DIR
+
+# Virtualenv activator
```

Comment:
This should always be the same relative to TURBINIAENV, so I think we can probably remove it, and then just do this same concatenation in the scripts using it.

Reply:
Removed